apps/x509.c

```diff
@@ -138,7 +138,9 @@ const OPTIONS x509_options[] = {
      "Number of days until newly generated certificate expires - default 30"},
     {"preserve_dates", OPT_PRESERVE_DATES, '-',
      "Preserve existing validity dates"},
-    {"subj", OPT_SUBJ, 's', "Set or override certificate subject (and issuer)"},
+    {"set_issuer", OPT_ISSU, 's', "Set or override certificate issuer"},
```

Comment:
why use set_ as the name?
It looks like it was done for 'serial' because it also has an increment

Reply:
Because there is already an `issuer` option `"issuer", OPT_ISSUER, '-', "Print issuer DN"`